tests/marginModels/tests

```diff
@@ -0,0 +1,17 @@
+[Tests]
```

Comment:
This whole file need to be updated, since tests are for Maintenance models

Reply:
fixed